src/NewTodo.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class NewTodo extends React.PureComponent {
+  state = {
+    users: this.props.user,
+    userID: 0,
+    taskDescription: '',
+    taskValidation: 1,
+    userValidation: 1,
+    value: '',
+  }
+
+  createList = arr => (
+    arr.map(user => (
+      <option key={user.email}>{user.name}</option>
+    ))
+  )
+
+  selectedUser = (event) => {
```

Comment:
name of the functions or methods must be verbs, so you need to rename it so `selectUser`

Reply:
fixed